src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -219,6 +219,8 @@ internal static SyntaxToken GetArgumentModifiers(RefKind refKind)
                     return SyntaxFactory.Token(SyntaxKind.OutKeyword);
                 case RefKind.Ref:
                     return SyntaxFactory.Token(SyntaxKind.RefKeyword);
+                case RefKind.RefReadOnlyParameter:
+                    return SyntaxFactory.Token(SyntaxKind.InKeyword);
```

Comment:
is this correct?  feels broken.  I also don't really understand why `'in` returns `default`.

Reply:
> is this correct?

Yes, arguments can be passed with `in` callsite modifier to `ref readonly` parameters (sometimes also with `ref` modifier, but not if they are read only; or without any callsite modifier but that's always a warning).